src/index.html

```diff
@@ -1,14 +1,33 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="container">
+    <header>
+      <a href="./images/logo.png" target="_blank"
+     rel="noopener noreferrer"
+     class="logo">
```

Comment:
Fix code style everywhere
```suggestion
      <a
        href="./images/logo.png"
        target="_blank"
        class="logo"
     >
```